### PR TITLE
Readme update usage section

### PR DESCRIPTION
**Pull request by ecbrodie**
This PR updates the README Usage section. I hope that this helps.

The main update is basically a partial revert of 52b4a13. I found it a little hard to follow the README as a new user of `babel-eslint` that I had to view the version compatibilities between ESLint and babel-eslint in two separate spots of the document. I am unsure whether this PR will conflict with the intent of the commit's author, @hzoo, but hopefully my point of view can be of value to help the experience of new `babel-eslint` users.

I appreciate the feedback. Thank you.

### CODE REVIEW

**Comment by ecbrodie** [comment]
Trying to close up loose ends on my open PRs. @hzoo do you have any update on the status of this PR? Feel free to close if you do not wish to accept this change. Thank you.

File: README.md [hunk 1]
```diff
@@ -43,20 +43,22 @@ It just needs to export a `parse` method that takes in a string of code and outp
...
+1.x | >= 5.x
 
 ### Install
 
+Ensure that you have substituted the correct version lock for `eslint` and `babel-eslint` into this command:
+
 ```sh
 $ npm install eslint@3.x babel-eslint@7 --save-dev
+# or
+$ yarn add eslint@3.x babel-eslint@y -D
```
Comment (hzoo) [inline README.md]: you mean 7 here right?

**Comment by hzoo** [comment]
Nah was on vacation at home, thanks!

**Comment by ecbrodie** [comment]
Thank you @hzoo 😄
